api/openapi.yaml

```diff
@@ -1056,10 +1056,18 @@ components:
         endTime:
           type: string
           format: date-time
+        timeoutTime:
+          description: |
+            A time in the future when this Scan will be automatically aborted
+            if it is still in progress.
+          type: string
+          format: date-time
         scanConfig:
           $ref: '#/components/schemas/ScanConfigRelationship'
+          readOnly: true
```

Comment:
This shouldn't be readOnly otherwise you can't set the `scanConfig: {id: ...}` field. The ScanConfigRelationship enforces that all fields except ID are readOnly.